NwPluginAPI/Enums/ServerEventType.cs

```diff
@@ -72,15 +74,15 @@ public enum ServerEventType : int
         /// Executed when grenade explodes.
         /// </summary>
         /// <remarks>
-        /// Parameters: <see cref="ItemPickupBase"/> pickup.
+        /// Parameters: <see cref="IPlayer"/> player, <see cref="Vector3"/> position, <see cref="ItemPickupBase"/> pickup.
```

Comment:
I think that using Footprint struct will be better than IPlayer. A player may leave after throwing the grenade. If that happens IPlayer becomes null and you have no way to determine who threw the grenade. That's why Footprint struct exists and is used by the base game in this event.

Reply:
The player can be null I don't think there are any problems (we could even give FootSprint and Player) in that what I would like to add would be an array with the people damaged by the grenade although I don't know if that would have to be divided in another event

Reply:
That array needs to be a separate event, because it's calculated after the grenade explodes. Having a separate event (one per player) will be better, because it will allow to selectively cancel events.